stdlib/pervasives.md

```diff
@@ -532,6 +532,32 @@ Returns:
 |----|-----------|
 |`Number`|The modulus of its operands|
 
+### Pervasives.**(**)**
```

Comment:
@spotandjake I think you found a bug with graindoc. Since `*` is bolding, this isn't shown correctly.

Reply:
f that is a good point, I guess ideally characters should be escaped.

Reply:
This was fixed by #1685 so this PR is unblocked now.